docs/README-windows.md

```diff
@@ -0,0 +1,125 @@
+opencv_world3411d.dll
+opencv_world3411d.pdb
+```
+
+**Alternatively**, one may add the path of OpenCV binary (eg: `D:\opencv\build\x64\vc15\bin`) to `PATH envionment variable`. You may refer to this [web page](https://www.deciphertechnic.com/install-opencv-with-visual-studio/).
+
+
+
+## Notes on install libwebp
+
+It locates within `Repo folder`, eg: `D:\splitwebp\3rdparty\libwebp\lib\windows-x64`
+
+Please note that, in Windows, you need to rename the files if you download libwebp from [here](https://storage.googleapis.com/downloads.webmproject.org/releases/webp/index.html):
+```
+libwebp.lib -> webp.lib
+libwebpdemux.lib -> webpdemux.lib
+libwebpmux.lib -> webpmux.lib
+```
+
+
+## Steps to compile
+
+1. Prepare the prerequisite
+
+2. Clone this repository to `Repo folder`, eg: `D:\splitwebp`
+
+3. Create a new folder named `build` under `Repo folder`, eg: `D:\splitwebp\build` (i.e. the `build folder`)
+
+4. Open `CMD`, change the current directory to `build folder`
+
+5. Run `cmake .. && cmake -DCMAKE_GENERATOR_PLATFORM=x64 --build .`
+
+6. Visit Windows Explorer in `build folder`. Open the Visual Studio Project by double clicking `splitwebp.vcxproj`
+
+7. Change Project properties (Solution Explorer > right click `splitwebp` > Properties)
+
+    7.1. pointing Include folder of OpenCV
+
+        In Properties, go to C/C++ > Additional Include Directories > click on the value and see a pull down menu > click Edit.. > click New Line. And add these paths:
+
+        (if your `opencv extrated folder` is `D:\opencv`)
+
+        D:\opencv\build\include
+        D:\opencv\build\include\opencv
+        D:\opencv\build\include\opencv2
+
+    7.2. remove `/machine:x86` to build for x64 platform (64 bit). Delete `/machine:x86` in Additional Option, and click Apply.
+        ![s](/docs/images/remove-x86.png)
+
+
+8. Build solution. click Visual Studio's menu > Build > Build Solution (F7)
+
+9. If no errors, the compiled exe file should be located at `Release` or `Debug` folder under the `build folder`, eg: `D:\splitwebp\build`
+
+
+## Troubleshoot
+
+1) > By not providing "FindOpenCV.cmake" in CMAKE_MODULE_PATH
+
+    Solution: set the `envionment variable` named `OPENCV_DIR`. Open `CMD`, and run this command (This may require Administrator right. ):
+
+    `setx -m OPENCV_DIR D:\opencv\build\x64\vc15\lib`
+
+2) > Cannot find or open the PDB file
+
+    is shown in the Output window of Visual Studio shows:
+
+    Solution: Turn on `Microsoft Symbol Servers` for debugging symbol. Please refer to this [web page](https://docs.microsoft.com/en-us/visualstudio/debugger/specify-symbol-dot-pdb-and-source-files-in-the-visual-studio-debugger?view=vs-2019) for further details.
+
+
+## Glossory
+- `CMD` Command Prompt in Windows, accept user's command. Run it by: Start Menu => search "cmd" => click "Command Prompt" OR Start Menu > "Run" > type "cmd" > press Enter
+
+- `Repo folder` The folder of this repository cloned into your local machine. eg: `D:\splitwebp`
```

Comment:
If you changed the "repo folder" to "splitwebp", then don't forget to delete this line :P